extensions/blocks/podcast-player/components/header.js

```diff
@@ -5,31 +5,29 @@ import { memo } from '@wordpress/element';
 
 const Header = memo(
 	( { playerId, title, cover, link, track, children, showCoverArt, showEpisodeDescription } ) => (
-		<div className="jetpack-podcast-player__header-wrapper">
-			<div className="jetpack-podcast-player__header" aria-live="polite">
-				{ showCoverArt && cover ? (
-					<div className="jetpack-podcast-player__track-image-wrapper">
+		<div className="jetpack-podcast-player__header">
+			<div className="jetpack-podcast-player__cover-wrapper" aria-live="polite">
```

Comment:
Is the Title part of the podcast cover? To me `cover` + `title` = `something else than cover`. Wondering if `cover` element fits well according to what it contains.

Reply:
it's a minor one, btw. Feel free to ignore it.

Reply:
Nice catch! I had the most difficulties with figuring out a name for this one 😄(and apparently I didn't make a very good job here 😬). How does `current-track-info` sound instead?

Reply:
> How does current-track-info sound instead?

Definitely like that one better than `cover-wrapper`, more descriptive 👌

Reply:
tricky 🤔My two cents:

From What I can understand, we have three sections according to their functionality: the "current track", the "player" and the "Playlist". And in this case, into the current track, there are: "header", which contains the "title" and "cover", and the track "description".

- Current Track
    - Header
        - Cover
        - Title
    - Description (body)
- Player
- Playlist
    - Track 01
    - Track 02
    - Track n

Wondering why the player is into the header.

it will require re-define the markup using the `current-track` in someplace there :-/

```html
<div class="current-track">
    <div class="header">
        <div class="cover"></div>
        <div class="title"></div>
    </div>
    <div class="description">
        Bla bla bla... because bla.
    </div>
</div>
```

Reply:
Renamed in https://github.com/Automattic/jetpack/pull/15211/commits/86036c2ecd4ee87e35e011e4f5b12834c759815d

Reply:
> Wondering why the player is into the header.

Do you mean design-wise? Current design shows the player element in the middle of the header, no?

Reply:
I mean in the markup. The player is at the same level that the description and the header. 

![image](https://user-images.githubusercontent.com/77539/78035120-0b6c4400-733f-11ea-9d30-336eba75e495.png)

Reply:
I guess it currently has to because of how the header is designed:
![image](https://user-images.githubusercontent.com/1451471/78037640-1be6e380-736c-11ea-9e3d-ef2f4d116fd3.png)